### DISCUSSION TITLE
I want to set a vertical reference line

### DISCUSSION THREAD

**Discussion by PILIHU2022** [LazyVim/LazyVim#1393] [Q&A] [answered]
# I want to set a vertical reference line to make it easier for me to write code, but I wrote vim.opt.colorcolumn = 80 in lua/config/options.lua, but it can’t take effect. Please how should I configure it to make it take effect?

**Comment by dpetka2001** [comment, marked as answer]
Try `vim.opt.colorcolumn = "80"`.

**Comment by PILIHU2022** [comment]
It works. Thanks. Have a nice day!

**Comment by dr-kernel** [comment]
vim.opt isn't available for me.  I get the error: `vim.opt.colorcolumn = "90" E480: No match: opt`
I added it to my `../lua/config/options.lua`

**Reply by dpetka2001** [reply]
You probably have an old version of Neovim.
